esridump/esri2geojson.py

```diff
@@ -1,18 +1,33 @@
 from itertools import tee
+from functools import partial
+from shapely.geometry import shape, mapping
+from shapely.ops import transform
+import pyproj
 
-def esri2geojson(esrijson_feature):
+def esri2geojson(esrijson_feature, srid = 'epsg:4326'):
     response = dict(type="Feature", geometry=None, properties=None)
 
     geojson_geometry = convert_esri_geometry(esrijson_feature.get('geometry'))
     if geojson_geometry:
+        if srid != 'epsg:4326':
```

Comment:
Hmm, I think it might be out of the scope of this tool to reproject. Usually we ask the server to give us the data back reprojected (with the `outSRS` parameter). Are you seeing servers that don't do that?

Reply:
@iandees Yeah the query API is the only endpoint that allows for `ourSRS` to be set. I played around with it a ton before I finally settled on having to do this. Not converting results in us outputting invalid GeoJSON per spec.